Mathlib/Algebra/Order/Antidiag/Pi.lean

```diff
@@ -1,17 +1,28 @@
 /-
 Copyright (c) 2023 Antoine Chambert-Loir and María Inés de Frutos-Fernández. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández, Eric Wieser, Bhavik Mehta
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández, Eric Wieser, Bhavik Mehta,
+  Yaël Dillies
 -/
-import Mathlib.Algebra.BigOperators.Fin
-import Mathlib.Algebra.Order.Antidiag.Prod
+import Mathlib.Algebra.Order.BigOperators.Group.Finset
+import Mathlib.Data.Finset.Pointwise
+import Mathlib.Data.Fin.Tuple.NatAntidiagonal
 
 /-!
 # Antidiagonal of functions as finsets
 
 This file provides the finset of functions summing to a specific value on a finset. Such finsets
 should be thought of as the "antidiagonals" in the space of functions.
 
+Precisely, for a commutative monoid `μ` with antidiagonals, `Finset.piAntidiag s n` is the finset of
```

Comment:
As in the other file, can you point to the main results, or main API things that a user might want to use when stumbling upon this file? And add a link to the definition of the `HasAntidiagonal` class?